tox.ini

```diff
@@ -6,12 +6,23 @@ deps =
     pytest
     pytest-cov
     itemadapter
-
+    sybil
+setenv =
+    PY_IGNORE_IMPORTMISMATCH=1
 commands =
     py.test \
         --cov-report=term-missing:skip-covered --cov-report=html --cov-report= --cov=zyte_common_items \
-        --doctest-modules \
```

Comment:
Just curious, what's the advantage of using sybil instead of --doctest-modules?

Reply:
I expected `--doctest-modules` to work right away, but it did not catch reStructuredText files after including `docs` in the command targets, so I started trying to find out why, reached https://docs.pytest.org/en/7.1.x/how-to/doctest.html#alternatives, failed to get the first suggested package, but managed to get Sybil working.

However, I did not read that page from the beginning, and I thought `--doctest-modules` could not support rst because of how they highlight the fact that the first suggested alternative supports rst. But I never tried `--doctest-glob="*.rst"`, featured at the very beginning of the page.

I do know that Sybil has some nice features, allowing e.g. to write hidden doctest content, not visible in the documentation but executed during tests. But I will try `--doctest-glob="*.rst"`, and if it works for us now, I think it best to use it and keep things simple.